### ISSUE TITLE
Adding folders named 'resources' to the path is not supported

### ISSUE THREAD

**Issue by d4n1elchen** [imatlab/imatlab#54]
I'm trying to run imatlab with MATLAB 2019a and Python 3.7. Every works fine currently except inline plot.

When I run the setup comman `imatlab_export_fig`, I got the following error:
```
Undefined function 'imatlab_export_fig' for input arguments of type 'char'.
```

I realized that `resources` folder wasn't loaded to path so I tried to add it manually but got the following error:
```
Warning: Adding folders named 'resources' to the path is not supported:
\path-to-Anaconda3\envs\ai-tutorial\lib\site-packages\imatlab\resources 
> In path (line 109)
  In addpath (line 86) 
```

I'm using conda to create isolated environments. I think this is a new problem in MATLAB 2019a because I run the same `addpath` command in 2018b and it executed successfully.

**Comment by anntzer** [comment]
Thanks for the report!
I moved `resources` to `data` :) please try master.

**Comment by d4n1elchen** [comment]
It works now! But is it normal that I need to `addpath` manually?

**Comment by anntzer** [comment]
No, it should be added by default (https://github.com/imatlab/imatlab/blob/master/lib/imatlab/_kernel.py#L164).  Can you do some print-debugging around this to check whether this line is correctly called and sets the path correctly?
Note that jupyter suppresses the output of the kernel, so for print-debugging you actually need to open a file, log to the file and manually inspect the result.

**Comment by d4n1elchen** [comment]
I found the problem. I change the environment, redo the installation but miss `python -mimatlab install`. After rerunning the command, everything works as normal.
